src/connect.js

```diff
@@ -20,7 +20,9 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  return function device(extraParams) {
```

Comment:
Good job!
better to write arrow function and use spread operator
```suggestion
  return (extraParams) => { ... params, ... extraParams }
```